libs/server.js

```diff
@@ -126,6 +125,7 @@ function startServer(){
 	naming.configure(miaou);
 
 	var cookieParser = require('cookie-parser')(miaou.config.secret);
+	var sessionStore = new RedisStore(miaou.config.redisStore);
```

Comment:
Please do `new RedisStore(miaou.config.redisStore || {})` in case the key wasn't provided. (This also has the advantage of being bc-compatible.)